src/main/java/org/jenkinsci/plugins/proccleaner/PsCleaner.java

```diff
@@ -92,7 +91,7 @@ public PsCleanerDescriptor() {
         }
 
         public String getUsername() {
-            return username;
+            return System.getProperty("user.name", "");
```

Comment:
of course I don't mean the remote call should be in inside getter, but whole setup needs to be called on the slave